test_deps.ts

```diff
@@ -0,0 +1,3 @@
+export * as asserts from "https://deno.land/std@0.116.0/testing/asserts.ts";
+export * as bench from "https://deno.land/std@0.116.0/testing/bench.ts";
+export * as expandGlob from "https://deno.land/std@0.116.0/fs/expand_glob.ts";
```

Comment:
```suggestion
export { expandGlob } from "https://deno.land/std@0.116.0/fs/expand_glob.ts";
```
There's no need to `export * as ... from` I think, it's possible to directly export a sub part of the lib.
It also avoids using `const {...} = ...lib` as you could just import them like it's currently:
```ts
import { expandGlob } from "../../test_deps.ts";
```